app/Livewire/Server/Navbar.php

```
@@ -61,14 +66,23 @@ public function restart()
-            StopProxy::run($this->server, restarting: true);
 
-            $this->server->proxy->force_stop = false;
-            $this->server->save();
+            // Prevent duplicate restart messages (e.g., from double-click or re-render)
+            if ($this->restartInitiated) {
+                return;
+            }
+            $this->restartInitiated = true;
+
+            // Always use background job for all servers
+            RestartProxyJob::dispatch($this->server);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Jobs\RestartProxyJob' in method 'restart'. (undefined)

(StaticAccess)